src/app/shared/components/template/components/toggle-bar/toggle-bar.html

```diff
@@ -0,0 +1,20 @@
+<div
+  class="container"
+  [class]="position"
+  [attr.data-param-style]="style"
+  [attr.data-variant]="variant"
+>
+  <div
+    *ngIf="!variantIsCardPortrait"
```

Comment:
nit(blocking) many of the same comments from the button component may apply here